test/Test/Cardano/Prelude/Tripping.hs

```diff
@@ -45,6 +47,24 @@ import Hedgehog.Internal.TH (TExpQ)
 import Language.Haskell.TH (Exp(..), Q, location, runIO)
 import Language.Haskell.TH.Syntax (Loc(..), mkName, unTypeQ, unsafeTExpCoerce)
 
+canonDecodePretty
```

Comment:
I'm not sure I would shorten `canonical` to `canon` but perhaps I'm nitpicking.

Reply:
I also think `canonical` would be better :)